namespaces/live-1.cloud-platform.service.justice.gov.uk/crime-portal-mirror-gateway-preprod/00-namespace.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: crime-portal-mirror-gateway-preprod
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "preproduction"
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "HMPPS"
+    cloud-platform.justice.gov.uk/application: "Crime Portal Mirror Gateway"
+    cloud-platform.justice.gov.uk/owner: "probation-in-court: christopher.faulkner@digital.justice.gov.uk"
```

Comment:
You should ideally use the team email here.
probation-in-court-team@digital.justice.gov.uk

Reply:
done